File: 2503-longest-subarray-with-maximum-bitwise-and/2503-longest-subarray-with-maximum-bitwise-and.java
class Solution {
    public int longestSubarray(int[] nums) {
       int max=nums[0],len=0,cnt=0;
       for(int i=1;i<nums.length;i++)
       {
            if(max<=nums[i])
            max=nums[i];
       }
        for(int n:nums)
        {
            if(n==max)
            {
                cnt++;
                len=Math.max(len,cnt);
            }
            else
            cnt=0;
        }
        return len;
    }
}

File: 2503-longest-subarray-with-maximum-bitwise-and/2503-longest-subarray-with-maximum-bitwise-and.py
class Solution:
    def longestSubarray(self, nums: List[int]) -> int:
        maxvalue=max(nums)
        count=0
        length=0
        for m in nums:
            if m==maxvalue:
                count+=1
                length=max(length,count)
            else:
                count=0
        return length